cli/src/index.ts

```diff
@@ -956,6 +956,324 @@ yargs(hideBin(process.argv))
+                        if (error instanceof Error) {
+                            console.error(error.message);
+                        } else if (error instanceof SendTransactionError) {
+                            console.error(error.logs);
+                        }
+                        process.exit(1);
+                    }
+                }
+
+                // use lut if configured
+                const luts: AddressLookupTableAccount[] = [];
+                try {
+                    luts.push(await solanaNtt.getAddressLookupTable());
+                } catch {}
+
+                // send versioned transaction
+                try {
+                    const latestBlockHash = await connection.getLatestBlockhash();
+                    const messageV0 = new TransactionMessage({
+                        payerKey: payerKeypair.publicKey,
+                        instructions: [
+                            await NTT.createAcceptTokenAuthorityInstruction(
+                                solanaNtt.program,
+                                await solanaNtt.getConfig(),
+                                {
+                                    currentAuthority: payerKeypair.publicKey,
+                                    multisigTokenAuthority,
+                                }
+                            ),
+                        ],
+                        recentBlockhash: latestBlockHash.blockhash,
+                    }).compileToV0Message(luts);
+                    const vtx = new VersionedTransaction(messageV0);
+                    vtx.sign([payerKeypair]);
+                    const signature = await connection.sendTransaction(vtx, {});
+                    await connection.confirmTransaction(
+                        {
+                            ...latestBlockHash,
+                            signature,
+                        },
+                        "finalized"
+                    );
+                    console.log(
+                        `Token mint authority successfully updated to ${(
+                            multisigTokenAuthority ?? tokenAuthority
+                        ).toBase58()}`
+                    );
+                } catch (error) {
+                    if (error instanceof Error) {
+                        console.error(error.message);
+                    } else if (error instanceof SendTransactionError) {
+                        console.error(error.logs);
+                    }
+                }
+            }
+            // undeployed case
+            else {
+                // verify new authority is valid SPL Multisig
+                if (multisigTokenAuthority) {
+                    // NOTE: the `acceptTokenAuthority` instruction would have done this check normally.
+                    // However, since the program is not yet deployed, we do this check here.
+                    const isMultisigTokenAuthority =
+                        await checkSolanaValidSplMultisig(
```

Comment:
If SPL Multisig works the same way for Fogo, we should probably rename the function to `checkSvmValidSplMultisig` to make it more generic.

```suggestion
                        await checkSvmValidSplMultisig(
```